include/fmt/chrono.h

```diff
@@ -1561,17 +1575,51 @@ template <typename OutputIt, typename Char> class tm_writer {
     write1(wday == 0 ? days_per_week : wday);
   }
 
-  void on_abbr_month() { format_localized('b'); }
-  void on_full_month() { format_localized('B'); }
+  void on_abbr_month() {
+    if (is_classic_)
+      out_ = write(out_, tm_mon_short_name(tm_mon()));
+    else
+      format_localized('b');
+  }
+  void on_full_month() {
+    if (is_classic_)
+      out_ = write(out_, tm_mon_full_name(tm_mon()));
+    else
+      format_localized('B');
+  }
 
   void on_datetime(numeric_system ns) {
-    format_localized('c', ns == numeric_system::standard ? '\0' : 'E');
+    if (is_classic_) {
+#ifdef _WIN32
+      on_us_date();
+      *out_++ = ' ';
+      on_iso_time();
+#else
```

Comment:
The locale-independent and C locale format should be consistent among platforms so let's remove this.

Reply:
Done. Corresponding tests for Windows are disabled (https://github.com/fmtlib/fmt/runs/4055476293?check_suite_focus=true)